apps/emqx_bridge_dynamo/test/emqx_bridge_dynamo_SUITE.erl

```diff
@@ -273,6 +275,24 @@ create_bridge_http(Params) ->
         Error -> Error
     end.
 
+update_bridge_http(#{<<"type">> := Type, <<"name">> := Name} = Config) ->
```

Comment:
Nit: there are `get` and `update` helper fns in `emqx_bridge_testlib`.

Reply:
Thanks! I will remember that next time! I don't think it will cause much harm to have a little bit of code duplication in a test suite so I will merge this now and maybe we can come back and fix this in a later PR.